segments/disk_usage.sh

```diff
@@ -0,0 +1,25 @@
+# Print used disk space on the specified filesystem
+
+TMUX_POWERLINE_SEG_DISK_USAGE_FILESYSTEM_DEFAULT="/"
+
+generate_segmentrc() {
+	read -d '' rccontents  << EORC
+# Filesystem to retrieve disk space information. Any from the filesystems available (run "df | awk '{print $1}'" to check them).
+export TMUX_POWERLINE_SEG_DISK_USAGE_FILESYSTEM="${TMUX_POWERLINE_SEG_DISK_USAGE_FILESYSTEM_DEFAULT}"
+EORC
+	echo "$rccontents"
+}
+
+run_segment() {
+	__process_settings
+    percentage=`df ${TMUX_POWERLINE_SEG_DISK_USAGE_FILESYSTEM} | awk '{print $5}' | tail -n1`
```

Comment:
Using $() is nice than ``, as it's visually easier to match a left parenthesis to a right one, and it's way clearer if you nest things

Reply:
oh, I know what you mean.
Will address this. Thanks for the recommendation.